### PR TITLE
Document new multi-asset-payment mode

### PR DESCRIPTION
**Pull request by laurencemcknight**
This PR only documents that the multi-asset-payment mode exists. 
Documentation about how to use multi-asset-payment will come in a follow up PR.

### CODE REVIEW

File: legacy/src/api/payment-requests/payment-requests.md [hunk 1]
```diff
@@ -898,19 +898,28 @@ Use authorization to authorize an external transaction.
...
-| authorization | String {% opt %}  | Used to authorize an external transaction.                     |
-| mode          | String {% opt %}  | The mode of payment valid value is `parital-payment`.          |
-| amount        | String {% opt %}  | The amount being paid if less than the Payment Request amount. |
+|     Field     |       Type       |                                    Description                                    |
+| :------------ | :--------------- | :-------------------------------------------------------------------------------- |
+| assetType     | String           | The name of the [Asset Type][].                                                   |
+| assetId       | String {% opt %} | The id of the Asset being used to make payment.                                   |
+| transactionId | String {% opt %} | Used to verify an external transaction eg Bitcoin.                                |
+| authorization | String {% opt %} | Used to authorize an external transaction.                                        |
+| mode          | String {% opt %} | The mode of payment. Valid value are `partial-payment` and `multi-asset-payment`. |
```
Comment (alexBinns) [inline legacy/src/api/payment-requests/payment-requests.md:921]:
```suggestion
| mode          | String {% opt %} | The mode of payment. Valid values are `partial-payment` and `multi-asset-payment`. |
```

**Review by alexBinns — APPROVED** [review]

**Comment by laurencemcknight** [comment]
To avoid confusion we are going to wait until multi asset payment is publicly available for integrators before documenting it. We are leaving this PR on ice until then.

**Comment by MeganSteenkamp** [comment]
@laurencemcknight just a heads up - we are going to finish our migration of the API documentation this week.
You'll likely need to change this PR to update the new docs if that's okay 🙌

**Comment by IsaacMurtagh** [comment]
@laurencemcknight , can you update this PR to add this to the new docs?

**Review by MeganSteenkamp — APPROVED** [review]

**Pull request merged**